arch/arm/core/cortex_m/mpu/arm_mpu_v7_internal.h

```diff
@@ -8,6 +8,11 @@
 #ifndef ZEPHYR_ARCH_ARM_CORE_CORTEX_M_MPU_ARM_MPU_V7_INTERNAL_H_
 #define ZEPHYR_ARCH_ARM_CORE_CORTEX_M_MPU_ARM_MPU_V7_INTERNAL_H_
 
+
+#define LOG_LEVEL CONFIG_ARCH_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_DECLARE(mpu);
```

Comment:
I think that even when #10045 is introduced this will not work. In arm_mpu.c (which includes this file) there will be 2 implementations of __log_current_const_data_get. We need to come up with a solution for logging in headers.